diff --git a/node_modules/@adobe/aio-lib-core-logging/src/DebugLogger.js b/node_modules/@adobe/aio-lib-core-logging/src/DebugLogger.js
index 90f47bb..31b6600 100644
--- a/node_modules/@adobe/aio-lib-core-logging/src/DebugLogger.js
+++ b/node_modules/@adobe/aio-lib-core-logging/src/DebugLogger.js
@@ -16,7 +16,7 @@ class DebugLogger {
     this.config = config
     debug.formatters.s = this.getFormat()
     debug.formatters.d = this.getFormat()
-    debug.log = this.getDestination()
+    // debug.log = this.getDestination()
     // make sure to always take into account the latest DEBUG env var
     debug.enable(process.env.DEBUG)
     if (debug.enabled(this.config.label)) {
